We don't do type checking (via mypy and related) downstream.
Mypy-protobuf generates mypy stub files from protobuf specs and should
therefore be stripped.
--- a/build/configure/src/python.rs
+++ b/build/configure/src/python.rs
@@ -87,9 +87,7 @@ pub struct GenPythonProto {
 impl BuildAction for GenPythonProto {
     fn command(&self) -> &str {
         "$protoc $
-        --plugin=protoc-gen-mypy=$protoc-gen-mypy $
         --python_out=$builddir/pylib $
-        --mypy_out=$builddir/pylib $
         -Iproto $in"
     }

@@ -107,7 +105,6 @@ impl BuildAction for GenPythonProto {
             .collect();
         build.add_inputs("in", &self.proto_files);
         build.add_inputs("protoc", inputs![":protoc_binary"]);
-        build.add_inputs("protoc-gen-mypy", inputs![":pyenv:protoc-gen-mypy"]);
         build.add_outputs("", python_outputs);
     }
 }
